ReactiveCocoa/Swift/Action.swift

```diff
@@ -134,7 +134,7 @@ public final class Action<Input, Output, Error: ErrorType> {
 public final class CocoaAction: NSObject {
 	/// The selector that a caller should invoke upon a CocoaAction in order to
 	/// execute it.
-	public static let selector: Selector = "execute:"
+	public static let selector: Selector = #selector(CocoaAction.execute(_:))
```

Comment:
Are the changes like these required for Swift 2.2? (Are they errors or just warnings?) They're breaking changes with Swift 2.1 / Xcode 7.2.1, so we'll need to release these as RAC 5.0 to follow semver.

(My Xcode download hasn't finished yet.)